bumble/host.py

```diff
@@ -660,7 +660,7 @@ def on_hci_le_remote_connection_parameter_request_event(self, event):
                 connection_handle=event.connection_handle,
                 interval_min=event.interval_min,
                 interval_max=event.interval_max,
-                latency=event.latency,
+                max_latency=event.max_latency,
```

Comment:
Yes! Somehow this fell through the cracks when I renamed `latency` to match the wording in the spec.

Reply:
what kind of bug would this be?

Reply:
`latency` was renamed with the `max_` prefix in both the event and command